### PR TITLE
[plugwiseha] Add and enhance channels

### PR DESCRIPTION
**Pull request by lsiepel**
1. Add Zone => cooling_allowed channel (r/w)
2. Add Zone => regulation_control channel (r/w)
3. Fix documentation mistake=> preheating_allowed channel is r/w
4. Changed Zone => scene channel from read-only to r/w.
5. Minor codestyle improvements

No configuration or other changes needed for the end-user.

### CODE REVIEW

File: bundles/org.openhab.binding.plugwiseha/src/main/java/org/openhab/binding/plugwiseha/internal/handler/PlugwiseHAZoneHandler.java [hunk 1]
```diff
@@ -186,6 +224,9 @@ protected void refreshChannel(Location entity, ChannelUID channelUID) {
                     state = new QuantityType<Temperature>(entity.getSetpointTemperature().get(), unit);
                 }
                 break;
+            case ZONE_REGULATION_CHANNEL:
+                state = new StringType(entity.getRegulationControl().orElse(null));
```
Comment (lolodomo) [inline bundles/org.openhab.binding.plugwiseha/src/main/java/org/openhab/binding/plugwiseha/internal/handler/PlugwiseHAZoneHandler.java:228]: Is StringType(null) a good state?

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Comment by lolodomo** [comment]
Default translations properties must be updated.

File: bundles/org.openhab.binding.plugwiseha/src/main/java/org/openhab/binding/plugwiseha/internal/handler/PlugwiseHAZoneHandler.java (hunk 1 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.plugwiseha/src/main/java/org/openhab/binding/plugwiseha/internal/handler/PlugwiseHAZoneHandler.java:228]: Yes, i think this is fine.

**Comment by lsiepel** [comment]
> Default translations properties must be updated.

I think i fixed them. Could you check? I'm new to these translations.

**Comment by lolodomo** [comment]
> I think i fixed them. Could you check? I'm new to these translations.

Just run : mvn org.openhab.core.tools:i18n-maven-plugin:3.2.0:generate-default-translations

**Comment by lsiepel** [comment]
> > I think i fixed them. Could you check? I'm new to these translations.
> 
> Just run : mvn org.openhab.core.tools:i18n-maven-plugin:3.2.0:generate-default-translations

Thanks! Got it sorted. I think all is good now.

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**